server/database/db_build.sql

```diff
@@ -0,0 +1,132 @@
+BEGIN;
+  DROP TABLE IF EXISTS places, guides, users, Arabic_words
+  CASCADE;
+create table places
+(
+  id serial primary key,
+  name varchar(100) not null,
+  type text not null,
+  image1 text not null,
+  image2 text,
+  image3 text,
+  descreption text,
+  availability text not null,
+  price text not null,
+  reviews text,
+  Address text not null
+);
+
+insert into places
```

Comment:
You shouldn't be adding so much data in the build file, this can be done in psql. What do you think @FarahZaqout?

Reply:
@jema28 resolve the comment and created a Heroku data base instead of locally.

Reply:
True, the build file should be only used to build the schema. Everything else must be in a different file.